### PR TITLE
Regenerate more stuff for grub 2.06

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

File: steps/grub-2.06/pass1.sh [hunk 1]
```diff
@@ -18,8 +18,14 @@ src_prepare() {
         patch -d grub-core/lib/gnulib -p2 < "grub-core/lib/gnulib-patches/$patchname.patch"
     done
 
+    # remove unauditable blobs (xz-style attack counter)
+    rm tests/dfly-mbr* tests/file_filter/*
```
Comment (Googulator) [inline steps/grub-2.06/pass1.sh:22]: Nice catch, thanks!

**Review by Googulator — COMMENTED** [review]
Any idea about the CI failures? Change on GitHub's side perhaps?

File: steps/grub-2.06/pass1.sh [hunk 2]
```diff
@@ -18,8 +18,14 @@ src_prepare() {
...
+
     ./autogen.sh
 
+    rm po/*.gmo po/exclude.pot
+    find . -name "*.info*" -delete
```
Comment (stikonas) [inline steps/grub-2.06/pass1.sh:27]: interestingly I've recently learned that `find . -name "*.info*" -delete` can be replaced by `find -name "*.info*" -delete` but oh well...

**Review by stikonas — APPROVED** [review]
Looks good though we need to sort out CI mess.

**Pull request merged**